pow/validator/src/main/java/org/ethereum/beacon/pow/validator/TransactionBuilder.java

```diff
@@ -0,0 +1,13 @@
+package org.ethereum.beacon.pow.validator;
+
+import org.ethereum.beacon.core.types.Gwei;
+import tech.pegasys.artemis.util.bytes.BytesValue;
+
+/** Util for creating validator transactions */
+public interface TransactionBuilder {
+  boolean isReady();
```

Comment:
Do we need an explicit check here? I think that a future which completes when sync is done would be sufficient enough for all interactions with Eth1. IMO, It should be handle on the higher abstraction layer than this.